### PR TITLE
Fix typ and add description

### PR DESCRIPTION
**Pull request by mkuzak**
Fixes issue #2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # 4oss-project
 
-## Second level heder
+This is an overrerview of the markdown syntax.
```
Comment (tobyhodges) [inline README.md]:
```suggestion
This is an overview of the markdown syntax.
```

Comment (tobyhodges) [reply README.md]: Please see corrected typo fix above

**Review by tobyhodges — CHANGES_REQUESTED** [review]
Please fix typo!

**Review by tobyhodges — APPROVED** [review]

**Comment by tobyhodges** [comment]
Thanks @mkuzak !

**Comment by mkuzak** [comment]
no worries, happy to help

**Pull request merged**